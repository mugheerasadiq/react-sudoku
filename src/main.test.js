import React from "react";
import renderer from "react-test-renderer";

import Main from "./main";

it("All props are passed", () => {
  const tree = renderer.create(<Main />).toJSON();
  expect(tree).toMatchSnapshot();
});
