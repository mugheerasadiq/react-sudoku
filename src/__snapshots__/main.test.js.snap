// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All props are passed 1`] = `
Array [
  <div
    className="full-page-loader"
  >
    <div
      className="ant-spin ant-spin-spinning"
    >
      <span
        className="ant-spin-dot ant-spin-dot-spin"
      >
        <i
          className="ant-spin-dot-item"
        />
        <i
          className="ant-spin-dot-item"
        />
        <i
          className="ant-spin-dot-item"
        />
        <i
          className="ant-spin-dot-item"
        />
      </span>
    </div>
  </div>,
  <div
    className="main_container"
  >
    <div
      className="main_wrapper"
    >
      <div
        className="header_container"
      >
        <h1
          style={
            Object {
              "color": "cadetblue",
              "fontSize": "42px",
              "fontWeight": "bolder",
            }
          }
        >
          suGOku
        </h1>
        <img
          height={150}
          src="logo.jpg"
          width={150}
        />
      </div>
      <table>
        <tbody>
          <tr>
            <td>
              <input
                name="0,0"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#90CAF9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="0,1"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#90CAF9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="0,2"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#90CAF9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="0,3"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#1DE9B6",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="0,4"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#1DE9B6",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="0,5"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#1DE9B6",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="0,6"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFAB91",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="0,7"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFAB91",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="0,8"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFAB91",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
          </tr>
          <tr>
            <td>
              <input
                name="1,0"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#90CAF9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="1,1"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#90CAF9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="1,2"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#90CAF9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="1,3"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#1DE9B6",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="1,4"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#1DE9B6",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="1,5"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#1DE9B6",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="1,6"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFAB91",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="1,7"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFAB91",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="1,8"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFAB91",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
          </tr>
          <tr>
            <td>
              <input
                name="2,0"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#90CAF9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="2,1"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#90CAF9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="2,2"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#90CAF9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="2,3"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#1DE9B6",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="2,4"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#1DE9B6",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="2,5"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#1DE9B6",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="2,6"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFAB91",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="2,7"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFAB91",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="2,8"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFAB91",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
          </tr>
          <tr>
            <td>
              <input
                name="3,0"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#D1C4E9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="3,1"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#D1C4E9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="3,2"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#D1C4E9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="3,3"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFF59D",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="3,4"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFF59D",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="3,5"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFF59D",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="3,6"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#A5D6A7",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="3,7"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#A5D6A7",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="3,8"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#A5D6A7",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
          </tr>
          <tr>
            <td>
              <input
                name="4,0"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#D1C4E9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="4,1"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#D1C4E9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="4,2"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#D1C4E9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="4,3"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFF59D",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="4,4"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFF59D",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="4,5"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFF59D",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="4,6"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#A5D6A7",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="4,7"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#A5D6A7",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="4,8"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#A5D6A7",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
          </tr>
          <tr>
            <td>
              <input
                name="5,0"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#D1C4E9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="5,1"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#D1C4E9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="5,2"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#D1C4E9",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="5,3"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFF59D",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="5,4"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFF59D",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="5,5"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#FFF59D",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="5,6"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#A5D6A7",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="5,7"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#A5D6A7",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="5,8"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#A5D6A7",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
          </tr>
          <tr>
            <td>
              <input
                name="6,0"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#80CBC4",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="6,1"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#80CBC4",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="6,2"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#80CBC4",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="6,3"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#F48FB1",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="6,4"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#F48FB1",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="6,5"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#F48FB1",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="6,6"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#81D4FA",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="6,7"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#81D4FA",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="6,8"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#81D4FA",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
          </tr>
          <tr>
            <td>
              <input
                name="7,0"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#80CBC4",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="7,1"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#80CBC4",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="7,2"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#80CBC4",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="7,3"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#F48FB1",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="7,4"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#F48FB1",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="7,5"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#F48FB1",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="7,6"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#81D4FA",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="7,7"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#81D4FA",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="7,8"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#81D4FA",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
          </tr>
          <tr>
            <td>
              <input
                name="8,0"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#80CBC4",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="8,1"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#80CBC4",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="8,2"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#80CBC4",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="8,3"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#F48FB1",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="8,4"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#F48FB1",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="8,5"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#F48FB1",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="8,6"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#81D4FA",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="8,7"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#81D4FA",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
            <td>
              <input
                name="8,8"
                onChange={[Function]}
                style={
                  Object {
                    "backgroundColor": "#81D4FA",
                    "fontSize": "3em",
                    "fontWeight": "150",
                  }
                }
                value=""
              />
            </td>
          </tr>
        </tbody>
      </table>
      <div
        className="difficulty_wrapper"
      >
        <h3
          className="ant-typography"
          onClick={null}
          style={
            Object {
              "WebkitLineClamp": undefined,
              "color": "cadetblue",
              "display": "inline",
              "marginRight": "15px",
            }
          }
        >
          Generate
        </h3>
        <button
          className="ant-btn ant-btn-default ant-btn-lg ant-btn-block"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#93ef00",
              "color": "white",
            }
          }
          type="button"
        >
          <span>
            Easy
          </span>
        </button>
        <button
          className="ant-btn ant-btn-default ant-btn-lg ant-btn-block"
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#e6da00",
              "color": "white",
            }
          }
          type="button"
        >
          <span>
            Medium
          </span>
        </button>
        <button
          className="ant-btn ant-btn-primary ant-btn-lg ant-btn-block ant-btn-dangerous"
          onClick={[Function]}
          type="button"
        >
          <span>
            Hard
          </span>
        </button>
        <button
          className="ant-btn ant-btn-primary ant-btn-lg ant-btn-block"
          onClick={[Function]}
          type="button"
        >
          <span>
            Random
          </span>
        </button>
        <button
          className="ant-btn ant-btn-default ant-btn-lg ant-btn-block ant-btn-dangerous"
          onClick={[Function]}
          style={
            Object {
              "marginLeft": "10px",
            }
          }
          type="button"
        >
          <span>
            Clear
          </span>
        </button>
      </div>
      <div
        className="footer_container"
      >
        <div
          className="footer_wrapper"
        >
          <div
            className="footer_btn"
          >
            <button
              className="ant-btn ant-btn-default ant-btn-lg ant-btn-block"
              onClick={[Function]}
              type="button"
            >
              <span
                aria-label="check"
                className="anticon anticon-check"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="check"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M912 190h-69.9c-9.8 0-19.1 4.5-25.1 12.2L404.7 724.5 207 474a32 32 0 00-25.1-12.2H112c-6.7 0-10.4 7.7-6.3 12.9l273.9 347c12.8 16.2 37.4 16.2 50.3 0l488.4-618.9c4.1-5.1.4-12.8-6.3-12.8z"
                  />
                </svg>
              </span>
              <span>
                Validate
              </span>
            </button>
            <input
              className="ant-input ant-input-sm"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              style={
                Object {
                  "height": "41px",
                }
              }
              type="text"
              value="Unsolved"
            />
          </div>
          <div
            className="footer_btn"
          >
            <input
              className="ant-input ant-input-sm"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              style={
                Object {
                  "height": "41px",
                  "textTransform": "capitalize",
                }
              }
              type="text"
              value="easy"
            />
            <button
              className="ant-btn ant-btn-default ant-btn-lg ant-btn-block"
              onClick={[Function]}
              type="button"
            >
              <span
                aria-label="container"
                className="anticon anticon-container"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="container"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M832 64H192c-17.7 0-32 14.3-32 32v832c0 17.7 14.3 32 32 32h640c17.7 0 32-14.3 32-32V96c0-17.7-14.3-32-32-32zm-40 824H232V687h97.9c11.6 32.8 32 62.3 59.1 84.7 34.5 28.5 78.2 44.3 123 44.3s88.5-15.7 123-44.3c27.1-22.4 47.5-51.9 59.1-84.7H792v-63H643.6l-5.2 24.7C626.4 708.5 573.2 752 512 752s-114.4-43.5-126.5-103.3l-5.2-24.7H232V136h560v752zM320 341h384c4.4 0 8-3.6 8-8v-48c0-4.4-3.6-8-8-8H320c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8zm0 160h384c4.4 0 8-3.6 8-8v-48c0-4.4-3.6-8-8-8H320c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8z"
                  />
                </svg>
              </span>
              <span>
                Difficulty
              </span>
            </button>
          </div>
        </div>
        <div
          className="footer_wrapper"
          style={
            Object {
              "marginBottom": "20px",
            }
          }
        >
          <button
            className="ant-btn ant-btn-default ant-btn-lg ant-btn-block"
            onClick={[Function]}
            style={
              Object {
                "backgroundColor": "#4CAF50",
                "color": "white",
                "height": "50px",
              }
            }
            type="button"
          >
            <span>
              Solve
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>,
]
`;
